curiouslearning/FeedTheMonsterJS
# Fix CR-64: Changed Amharic forder name into lowercase

# Changes
- Amharic forder name into lowercase

# How to test
- Check the folder and file naming convention.

Ref: [CR-64](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-64)


[CR-64]: https://curiouslearning.atlassian.net/browse/CR-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
